applications/serial_lte_modem/prj_secure_bootloader.conf

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_SECURE_BOOT=y
+#CONFIG_SB_SIGNING_KEY_FILE="my_key_file.pem"
```

Comment:
Why is this commented out?

Reply:
This is for customer to specify their own key to be used by Nordic immutable bootloader.
By default we use the default key in NCS, like that used in MCUBOOT